cns/restserver/internalapi.go

```diff
@@ -389,3 +389,61 @@ func (service *HTTPRestService) CreateOrUpdateNetworkContainerInternal(req *cns.
 
 	return returnCode
 }
+
+// GetHomeAz gets home AZ from cache or from nmagent
+func (service *HTTPRestService) GetHomeAz(ctx context.Context) cns.GetHomeAzResponse {
+	homeAz := service.readHomeAzCache()
+	if homeAz != "" {
+		return cns.GetHomeAzResponse{HomeAzResponse: nmagent.HomeAzResponse{HomeAz: homeAz}}
+	}
```

Comment:
As discussed today, this func will only check the cache and return the value and cached NMA response